src/containers/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -133,6 +144,14 @@ const Breadcrumbs = () => {
   return (
     <Styled.Wrapper>
       <Styled.Crumbtainer>
+        {uriDisplay && localUri && (
+          <Button
```

Comment:
Can this button but part of the breadcrumbs so when hovering both are highlighted at the same time.

![image](https://github.com/user-attachments/assets/6eba58b2-8466-4895-a126-0c28531bc60a)